### PR TITLE
Change default region to us-east-1 and update schema with

### LINKED ISSUE #465: change default aws region to us-east-1
default region is currently us-east-2 for keeping CI stuff siloed.  However, this means as people on our team regenerate their configs they will end up in the CI region.

We should:
- change the default to us-east-1 (or us-west-2, either one)
- alter the CI process to use us-east-2 instead by re-writing the config file after generation
  - just a couple of sed one-liners will do

### PR DESCRIPTION
**Pull request by davidewatson**
documentation and defaults. Fixes #465.

### CODE REVIEW

File: ansible/roles/kraken.config/files/config.yaml [hunk 1]
```diff
@@ -302,19 +302,19 @@ definitions:
       type: cloudinit
       resourcePrefix:
       vpc: 10.0.0.0/16
-      region: us-east-2
+      region: us-east-1
       subnet:
         -
           name: zone-1
-          az: us-east-2a
+          az: us-east-1a
```
Comment (coffeepac) [inline ansible/roles/kraken.config/files/config.yaml]: us-east-1a is not available to us.  its a legacy only zone.

Comment (davidewatson) [inline ansible/roles/kraken.config/files/config.yaml]: I learned today that this is no longer true: "To ensure that resources are distributed across the Availability Zones for a region, we independently map Availability Zones to identifiers for each account."

Comment (coffeepac) [inline ansible/roles/kraken.config/files/config.yaml]: well I'll be.  Just checked the instances from the first run and there are instances in us-east-1.  

sweet.  I can stop moving everything out of us-east-1a when I'm moving a cluster from us-west-2 to us-east-1

File: build-scripts/update-generated-config.sh [hunk 2]
```diff
@@ -7,8 +7,9 @@ set -x
...
-sed -i -e "s/cluster:/cluster: ${cluster_name}/" $1
-
 #  new style config
-sed -i -e "s/- name:[[:space:]]*$/- name: ${cluster_name}/" $1
\ No newline at end of file
+sed -i -e "s/- name:[[:space:]]*$/- name: ${cluster_name}/" $1
+
+# move regions and AZs to us-east-2. note that this is the CNCT CI region for
+# API rate limit purposes.
+sed -i -e "s/us-east-1/us-east-2/g" $1"
```
Comment (coffeepac) [inline build-scripts/update-generated-config.sh]: there is a trailing double quote on the end of this line.  its causing the script to fail in CI:  https://common-jenkins.kubeme.io/job/K2/view/change-requests/job/PR-510/3/execution/node/26/log/

**Comment by coffeepac** [comment]
failed to destroy GKE cluster.  all other steps passed and nodes ran in us-east-2

**Pull request merged**